shared/fs/common/filename.js

```diff
@@ -38,7 +38,7 @@ const Filename = (props: Props) => {
       >
         {fileNameWithoutExtension}
       </Kb.Text>
-      {fileExtension && (
+      {!!fileExtension && (
```

Comment:
@jakob223 sorry for missing this in review. Problem here is when `fileExtension` is an empty string, it'd just end up in there, and on Android strings have to be inside the `Text` component.

Reply:
oops 😬